meta-resin-common/classes/image_types_resin.bbclass

```diff
@@ -264,3 +264,14 @@ IMAGE_CMD_docker () {
     DOCKER_IMAGE=$(${IMAGE_CMD_TAR} -cv -C ${IMAGE_ROOTFS} . | docker import -)
     docker save ${DOCKER_IMAGE} > ${IMGDEPLOYDIR}/${IMAGE_NAME}${IMAGE_NAME_SUFFIX}.docker
 }
+
+IMAGE_TYPEDEP_hostapp-ext4 = "docker"
+
+IMAGE_DEPENDS_hostapp-ext4 = " \
+    mkfs-hostapp-native \
+    "
+
+IMAGE_CMD_hostapp-ext4 () {
+    dd if=/dev/zero of=${IMGDEPLOYDIR}/${IMAGE_NAME}${IMAGE_NAME_SUFFIX}.hostapp-ext4 seek=$ROOTFS_SIZE count=0 bs=1024
```

Comment:
I guess this needs to accommodate older boards which don't use a yocto version that does not use IMGDEPLOYDIR as the variable designating the image deploy directory.

Reply:
I'm not sure how to do this. Can you give some more info?

Reply:
This is how we handle this: https://github.com/resin-os/meta-resin/blob/master/meta-resin-common/classes/image-resin.bbclass#L15

Reply:
Yes. This needs to accommodate both variables.